### DISCUSSION TITLE
How to best deal with losing text focus when resizing browser windows

### DISCUSSION THREAD

**Discussion by h43z** [i3/i3#5687] [Q&A]
Tiling window manager users probably do a lot more resizing, toggle fullscreen'ing than floating wm users. How do you deal with following situation. 

* you are reading a wikipedia article in your browser window (it's the only window open on your workspace)
* you are opening another window (say a terminal, you want to take notes)
* text reflows on the wikipedia article (chaos, you don't know where you finished reading)
* you are totally lost (you start crying)

How to not cry?

**Comment by okraits** [comment]
I use `workspace_layout tabbed`. So I don't have that issue. If I want to see two windows side by side, I either change to split layout or I move the window out of the tabbed container.

**Reply by h43z** [reply]
You never go back and forth between fullscreen/half(or whatever) screen on a browser window?

**Reply by okraits** [reply]
Not often and mostly for videos. And the browser is in a fullscreen tabbed container most of the time, so it doesn't make much difference.

**Comment by h43z** [comment]
If anyone is interested I created a browser extension (just 13 lines of code) that tries to correct the scroll position after window resizing. 

https://github.com/h43z/scrollcorrect-extension
